package-lock.json

```diff
@@ -0,0 +1,5650 @@
+{
```

Comment:
Don't include `package-lock.json`, `yarn` is used for this project

Reply:
I'll probably update `.gitignore` to ignore this file. But should still use `yarn` when contributing so we're working on same lock-files. My bad for not having Contribution-guides for this